fintorch/cli.py

```diff
@@ -30,7 +34,49 @@ def train(model):
     """Train financial models"""
     # Implement your model training logic here
     click.echo(f"Training model: {model}")
+    cli = LightningCLI(
```

Comment:
Skip this, and make this a sweep command for a particular model that is trained on a dataset.